### ISSUE TITLE
Forms do not scale when viewport is too small

### ISSUE THREAD

**Issue by ImhChr** [smathot/osweb#97]
Hi,

I've recently discovered the issue that certain page contents are inaccessible if the window isn't maximised and/or the user is zoomed in too much. Scrolling up isn't possible either. However, maximising the window and making sure the zoom doesn't exceed 100% solves the problem. After playing around in the settings I discovered that the style setting "display: flex" is to blame. If I disable it in the DevTools, scrolling up to the previously hidden contents works fine.

From what I can gather, this setting cannot be overridden in the script. I assume there is a good reason "flex" is the default, but maybe there is a way to make it more flexible? (oh, the irony)

Thank you in advance!

Cheers,
Chris

**Comment by smathot** [comment]
Thanks for reporting this!

**Comment by ImhChr** [comment]
Hi @smathot,

Thank you for addressing this issue! I've just updated to OSWeb 1.4.12, but the issue still persists...


Cheers,

Chris

**Comment by smathot** [comment]
Can you show a screenshot of what you mean exactly? The size of the forms should now scale, however the contents (font size, etc.) should not.

**Comment by ImhChr** [comment]
Of course :)

I've attached two screenshots. In the first one, I didn't maximise the window, resulting in the top part of the form being cut off, without the possibility to scroll up any further. In screenshot #2, I enlarged the window, with the whole page being visible now.

![Screenshot_OSWeb_Issue97_1](https://user-images.githubusercontent.com/40172312/173574045-95fc65b1-6393-4e96-8e00-e2c6ae4a6f47.PNG)
![Screenshot_OSWeb_Issue97_2](https://user-images.githubusercontent.com/40172312/173574049-ac7b7d53-1e39-4d5e-94b4-f1b972c7a8d7.PNG)
